topi/recipe/rnn/lstm.py

```diff
@@ -1,4 +1,6 @@
 """LSTM Example, still work in progress.."""
+from __future__ import division
```

Comment:
you don't need to import this if you are not using `/` anymore

Reply:
This is used some where
https://github.com/dmlc/tvm/pull/2131/files#diff-08e32b39e652ceee87d26c679fb8d747R46

Reply:
in the link you posted that's floor div `//`, which is supported in Python2

Reply:
I get what you mean...okay, i will remove the division thing.